Makefile

```diff
@@ -66,8 +66,13 @@ help:
 	@echo "- ci                       Create the python virtual environment and install requirements based on the Pipfile.lock"
 	@echo -e " \033[1mFORMATING, LINTING AND TESTING TOOLS TARGETS\033[0m "
 	@echo "- format                   Format the python source code"
+	@echo "- ci-check-format          Format the python source code and check if any files has changed. This is meant to be used by the CI."
 	@echo "- lint                     Lint the python source code"
 	@echo "- test                     Run the tests"
+	@echo -e " \033[1mSPEC TARGETS\033[0m "
+	@echo "- lint-spec                Lint the openapi spec"
+	@echo "- build-spec               Build the openapi spec"
```

Comment:
S : maybe .PHONY build-spec instead of build-spec**s**

Reply:
Actually plural is more correct since we have 2 specs, openapi.yaml and openapitransactional.yaml. I corrected to have all specs target in plural.